### PR TITLE
Release 0.7.0

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -11,9 +11,9 @@ Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
-JuMP = "0.19, 0.20, 0.21"
+JuMP = "0.21"
 MathOptInterface = "0.9"
-NLPModels = "0.12, 0.13"
+NLPModels = "0.13"
```
Comment (dpo) [inline Project.toml:16]: Are those incompatible now?

File: Project.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name = "NLPModelsJuMP"
 uuid = "792afdf1-32c1-5681-94e0-d7bf7a5df49e"
-version = "0.6.3"
```
Comment (abelsiqueira) [inline Project.toml:3]: If we are dropping support, we need to update to "0.7.0"

File: Project.toml (hunk 1 above)
Comment (amontoison) [reply Project.toml:16]: Partially, https://github.com/JuliaSmoothOptimizers/NLPModels.jl/pull/288 was only added at the last minor release of NLPModels 0.12.x.

**Comment by abelsiqueira** [comment]
Thanks

**Pull request merged**